idutils/__init__.py

```diff
@@ -181,6 +181,10 @@
 """Matches new style arXiv ID, with an old-style class specification;
     technically malformed, however appears in real data."""
 
+hal_regexp = re.compile("(hal:)?([a-z]{3}[a-z]*-|(sic|mem|ijn)_)\d{8}$")
+""" Matche HAL identifiers
```

Comment:
is it possible that hal: will appear as uppercase? f.e. like in ads_regexp below?
minor: typo in docstring